### ISSUE TITLE
Can not render the math equation 

### ISSUE THREAD

**Issue by rfhits** [futurepress/epub.js#1301]
the book: "Fundamentals-of-Computer-Graphics_5th-edition.epub"

the equation in *2.1.3 Logarithms*:

```latex
y = \log_{a}{x}
```
rendering result of epub.js:

![image](https://user-images.githubusercontent.com/70683664/201859542-afadf93b-62c5-4ed2-a61f-9e930b1e0800.png)

should be rendered as:

![image](https://user-images.githubusercontent.com/70683664/201859717-4578e7e1-5ed2-4051-af97-1521b2b11ded.png)

the size of this epub file is 32MB, so I put it in the google dirve: https://drive.google.com/file/d/1snuFibTRnWjLmjUM_q_jKSJ5OG8Rx7_v/view

**Comment by johnfactotum** [comment]
When using a browser engine that doesn't support MathML yet (i.e. Chromium), you need to use something like MathJax. See https://github.com/futurepress/epub.js/blob/master/examples/mathml.html.

**Comment by rfhits** [comment]
I tried. I replace the book url, but the math equation remains the same.

**Comment by johnfactotum** [comment]
The example is broken. It needs to be updated to set `allowScriptedContent` to `true`. See https://github.com/futurepress/epub.js/blob/master/README.md#scripted-content

**Comment by rfhits** [comment]
not work.

I notice that some requests were sent by the mathjax, does it matters?

my folder:
```
. 
|- ajax-loader.gif
|- epub.js
|- epubjs-mathml.html
|- examples.css
 Fundamentals-of-Computer-Graphics_5th-edition.epub

```

![image](https://user-images.githubusercontent.com/70683664/201915665-bde81292-8b12-4f0b-b613-e0b06b7dae35.png)

**Comment by johnfactotum** [comment]
> I notice that some requests were sent by the mathjax, does it matters?

Yeah, if you're using the CDN then it should be fetching resources from the CDN, not 127.0.0.1:5500. I don't know what could be causing this, though.

But at least the `examples/mathml.html` page works if you clone Epub.js, run npm install, and open the page directly in the browser. So in your case, probably either MathJax isn't included properly or some additional configuration is required for it to resolve resources correctly.
